_topic_maps/_topic_map.yml

```diff
@@ -1460,6 +1460,8 @@ Topics:
     File: persistent-storage-csi-cloning
   - Name: CSI automatic migration
     File: persistent-storage-csi-migration
+  - Name: CSI driver volume detach with non-graceful node shutdown
```

Comment:
I think we should rename that feature. "Detach CSI volumes after a non-graceful node shutdown" ?

Reply:
Done.